### DISCUSSION TITLE
Feature Request: Button to go to "Next" and "Previous" day

### DISCUSSION THREAD

**Discussion by chrisl8** [Freika/dawarich#296] [Ideas]
I'd like for there to be buttons to enable just walking back or forward one day at a time from whatever day is displayed at the moment.

I realize this is confusing if more than a day is selected, so I'm not exactly sure what the best user interface setup would be.

Basically though, I want to be able to open the app, and instead of only being able to select "Yesterday" and then having to use the calendar input boxes to get to days, I want to be able to just click "Previous Day" and "Next Day" to page through days one at a time.

I could see this being expanded to doing it by weeks and months also, but again, I'm not sure yet how the user interface would look. If I could imagine it I'd make a mock-up, but nothing has come to mind so far.

**Comment by mcridermca** [comment]
I second this request.

My use case requires reviewing my map once a month for invoicing, and need to walk every day 1 by 1 to track confirm my stops. A next an previous button would be a beautiful addition.

Keep up the work, this is an amazing project!

**Comment by tabacha** [comment]
Perhabs we can use:
https://www.npmjs.com/package/bootstrap-daterangepicker or
https://www.daterangepicker.com/

**Comment by comdoxx** [comment]
Oh! And while we're talking about "days". Google magically splits days conveniently. When I leave a bar far after midnight (lets say already sunday morning) to head home, this (not only the bar but also going home and home as final destination) appears on Saturday. Sunday then starts from home. So the day might be either static like from 3 am to 3 am or might even magically "breathe".

**Reply by marsmathis** [reply]
Yes, having this as an option would be quite nice. Maybe open a separate discussion topic for this?

**Comment by Freika** [comment]
Done in 0.26.1
